less/forms.less

```diff
@@ -22,7 +22,8 @@ select {
   width: 100%;
 }
 
-select.form-control {
+select.form-control,
+.bookmarkForm select {
```

Comment:
I know this is not an elegant solution; general properties for `select` and `input` should be specified at first, followed by more specific properties for `.form-control`, etc.